### ISSUE TITLE
[Settings]: Screen Reader is not announcing selected value name for 'Default terminal application' combo-box.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#11251]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Repro Steps:**

1. Open Windows Terminal.
2. Open Settings page using "Ctrl + ',' ".
3. Open Narrator using 'Win + Ctrl + Enter' key.
4. Navigate to Startup tab and select it then navigate to 'Default terminal application' combo-box. Observe the issue.

**User Experience:**
Screen Reader user are impacted here as they will not be able to know which options is selected on that combo-box and there is only one item in combo-box so no other to find what item is selected.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[Screen Reader is not announcing selected value name for 'Default terminal application' combo-box..zip](https://github.com/microsoft/terminal/files/7177816/Screen.Reader.is.not.announcing.selected.value.name.for.Default.terminal.application.combo-box.zip)

### Expected Behavior

Screen Reader should announce 'Default terminal application Windows Console Host Microsoft Corporation comb-box collapsed The application that launches...'

### Actual Behavior

Screen Reader is not announcing selected value name for 'Default terminal application' combo-box. Issue both repro in Narrator Scan on and Scan off mode. Currently Screen Reader is announcing 'Default terminal application comb-box collapsed the application that launches...'

Observation: Same issue repro with NVDA Screen Reader.

**Comment by carlos-zamora** [comment]
@v-rpundir  This is now fixed in v1.12.10334.0. Can you please confirm?

**Comment by v-rpundir** [comment]
Hi @carlos-zamora the latest build I have is Terminal Preview 1.12.3472.0 so I will keep this work in my To-Do list for now and will check when mentioned build is available here.

It will be better if you can provide us mentioned build us to verify the same.

**Comment by v-rpundir** [comment]
Closing the bug as it is fixed on Windows Terminal Preview Version: 1.13.10336.0
